addons/web_hierarchy/static/src/hierarchy_view.scss

```
@@ -0,0 +1,11 @@
+.o_hierarchy_icon { // button in view switcher
```

```suggestion
.o_control_panel_navigation .o_switch_view .o_hierarchy_icon { // button in view switcher
```
as the classname is quite generic

---

Hello @aab-odoo ,

Thanks for your review, I did the changes. 🙂